sunpy/timeseries/sources/fermi_gbm.py

```diff
@@ -59,14 +59,16 @@ class GBMSummaryTimeSeries(GenericTimeSeries):
     # Class attribute used to specify the source class of the TimeSeries.
     _source = 'gbmsummary'
 
-    def plot(self, axes=None, **kwargs):
+    def plot(self, axes=None, column=None, **kwargs):
```

Comment:
The original issue says its `columns` and not `column`, can you check which one is corrrect?

Reply:
In the title of the issue it is `column`.

Reply:
Can you check this again, what if the title was wrong?